3.1/imports/model.md

```diff
@@ -44,6 +44,24 @@ public function model(array $row)
 }
 ```
 
+## Possible column names
+
+In case you want to import rows by several possible names, you can use `??` operator. If the column with the first name (in example _client_name_) exists and is not NULL, return its value; otherwise look for second possible name (in example _client_) etc.
```

Comment:
> several possible names

several possible column names (using `WithHeadingRow`)

> you can use `??` operator

you can use null coalescing operator (`??`).